src/core/styles/ThemeProvider.tsx

```diff
@@ -0,0 +1,14 @@
+import {
+  ThemeProvider as RawThemeProvider,
+  ThemeProviderProps,
+} from "@material-ui/core/styles";
+import React from "react";
+import defaultTheme from "./common/defaultTheme";
+
+const ThemeProvider = (props: ThemeProviderProps) => {
+  const { theme } = props;
+
+  return <RawThemeProvider {...props} theme={theme || defaultTheme} />;
+};
+
+export default ThemeProvider;
```

Comment:
Why define our own `ThemeProvider`? Where will this be used?  If we use it here why not ask consumers to use it in our README

Reply:
ah thank you! This file is now deleted 😆